Dockerfile

```diff
@@ -54,16 +55,16 @@ RUN : \
   && rm /tmp/packages-microsoft-prod.deb \
   && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
   && echo "deb [arch=amd64] https://download.docker.com/linux/debian ${VERSION_CODENAME} stable" >> /etc/apt/sources.list \
-  && curl -fsSL https://packages.erlang-solutions.com/debian/erlang_solutions.asc | apt-key add - \
-  && echo 'deb https://packages.erlang-solutions.com/debian bullseye contrib' >> /etc/apt/sources.list \
+  && curl -fsSL https://binaries2.erlang-solutions.com/GPG-KEY-pmanager.asc | apt-key add - \
+  && echo "deb https://binaries2.erlang-solutions.com/debian bullseye-elixir-1.15 contrib" >> /etc/apt/sources.list \
```

Comment:
can you do this in a separate PR so it's clear why it happened and easier to revert if needed?

Reply:
Yeah makes sense, let me split this

Reply:
https://github.com/getsentry/craft/pull/577